cloud_edition/flexibility_mode/docs/disk_usage.rst

```diff
@@ -32,13 +32,34 @@ The disk usage will increase following the lifecycle of the instance. For exampl
 
 Impact of Disk Usage on the PIM
 -------------------------------
+
 When the disk is full, the PIM cannot work anymore: it cannot save modifications in database, upload new assets, ...
 
 It can even prevent ElasticSearch from updating its indexes, resulting in desynchronization with the database.
 To avoid those issues, ensuring a healthy disk usage is necessary.
 
+Check Disk Space Usage
+----------------------
+
+The following command can help you to check the disk space usage expressed both
+in GB and as a percentage:
+
+.. code-block:: bash
+
+    echo "Report disk space usage for my instance"
+    df -h --type ext4
+      Filesystem      Size  Used Avail Use% Mounted on
+      /dev/sdb        147G  106G   34G  76% /
+
+As explained in the previous paragraph, the disk mounted on "/" must not be
```

Comment:
Not only the '/'
on flex 8.x the data disk is /data

Reply:
I agree with Seb, as the disk space's user has bought is mounted on `/data` it would be simpler to monitor with only this one in the command.